plugins/versionpress/tests/Unit/QueryLanguageUtilsTest.php

```diff
@@ -101,8 +101,6 @@ public function ruleAndQueryProvider()
     public function queryLanguageUtilsCreatesCorrectRules($query, $expectedRules)
     {
         $rules = QueryLanguageUtils::createRulesFromQueries($query);
-        // Perform case insensitive match
-        $this->assertEquals($expectedRules, $rules, '', 0, 10, false, true);
```

Comment:
Ugh, I must have been sleeping already, this removes not only the case sensitivity hack but also the assert itself!

<img width="300" src="https://user-images.githubusercontent.com/101152/57836436-fb984f80-77c0-11e9-8d3a-ab4dd6423e5f.png">

Reply:
Resolved by 4b4353cd6ab90e913b5fb5b4db9b782b1bb1824a.